e2e-tests/arbiter/compare/statefulset_arbiter-rs0-arbiter.yml

```diff
@@ -87,6 +87,12 @@ spec:
                 - /data/db/mongodb-healthcheck
                 - k8s
                 - liveness
+                - --ssl
+                - --sslInsecure
+                - --sslCAFile
+                - /etc/mongodb-ssl/ca.crt
+                - --sslPEMKeyFile
+                - /tmp/tls.pem
```

Comment:
This files depend on secret existence.
But secret not always can exists. And if is doesn't, readiness and liveness probe will fail.

Reply:
fixed as a part of d7755107